src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/osd/osd-list/osd-list.component.html

```diff
@@ -6,34 +6,11 @@
           [updateSelectionOnRefresh]="false">
   <div class="table-actions btn-toolbar"
        *ngIf="permission.update">
-    <div class="btn-group"
-         dropdown>
-      <button dropdownToggle
-              type="button"
-              class="btn btn-sm btn-primary dropdown-toggle tc_scrub_toggle"
-              [ngClass]="{disabled: !tableComponent.selection.hasSelection}">
-        <ng-container i18n> Perform Task</ng-container>
-        <span class="caret"></span>
-      </button>
-      <ul *dropdownMenu
-          class="dropdown-menu"
-          role="menu">
-        <li role="menuitem"
-            (click)="scrubAction(false)">
-          <a>
-            <i class="fa fa-fw fa-stethoscope"></i>
-            <ng-container i18n>Scrub</ng-container>
-          </a>
-        </li>
-        <li role="menuitem"
-            (click)="scrubAction(true)">
-          <a>
-            <i class="fa fa-fw fa-stethoscope"></i>
-            <ng-container i18n>Deep Scrub</ng-container>
-          </a>
-        </li>
-      </ul>
-    </div>
+    <cd-table-actions [permission]="permission"
```

Comment:
Missing `.btn-group` on this element.

Reply:
After looking at it again, the problem should not be this class.
But the rendered element has some problems.

Reply:
Before:
![image](https://user-images.githubusercontent.com/399326/45682812-5cd82800-bb39-11e8-919b-6c4acb28d55b.png)
After:
![image](https://user-images.githubusercontent.com/399326/45682756-30241080-bb39-11e8-975d-246b0d450386.png)

Reply:
The class was missing. In your screenshot the button is disabled because all actions are disabled, but I don't like this approach, because this way, you cannot see which options you have. Leaving it enabled with disabled actions is more user friendly and more common as we are doing this in all other lists, too.